UtopiaCity/Models/Airport/TransportManagerSystem/Enums/TypeOfTransportForPassengers.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace UtopiaCity.Models.Airport.TransportManagerSystem.Enums
+{
+    public enum TypeOfTransportForPassengers
+    {
+        Car = 0,
+        BusNumber4 = 1,
```

Comment:
I would say that busnumber should not be a enum
Bus can be - then we can specify number in some field

Reply:
ok i will change